### ISSUE TITLE
[Bug] [a11y] The "+" and "+PRIVATE" buttons have the old UI/aren't displayed at all when talkback/voice assistant is enabled

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#14946]
## Steps to reproduce
1. Activate voice assistant or talkback.
2. Open Fenix Nightly, open a few tabs and observe the "Open new tab" or "Open new private tab" buttons.

### Expected behavior
The options to open a new tab or a new private tab are visible and can be used.

### Actual behavior
The options to open a new tab or a new private tab disappear, sometimes are replaced with a "+" button in from of the three-dot menu, sometimes aren't displayed at all, the user cannot open a new tab/a new private tab.

### Device information
* Android device: HTC 10 (Android 8), Samsung Galaxy Tab S6 (Android 9)
* Fenix version: RC 80.1.2, 9/10 Nightly, 81.1.0-beta.2

<img src="https://user-images.githubusercontent.com/40694158/92736521-0518d400-f383-11ea-86df-e310da3bc10b.gif" width="300" height="550" />

**Comment by abhijitvalluri** [comment]
Note that this issue only occurs in light theme. Users who use dark theme will not see this issue. So to make sure you reproduce the issue, ensure that you are using light theme.

**Comment by softvision-miralobontiu** [comment]
This issue still reproduces on Beta 81.1.1-beta.3, with Samsung Galaxy Tab S6 (Android 9).

**Comment by mcarare** [comment]
Having the tab + button instead of the floating one is the expected behavior, introduced in https://github.com/mozilla-mobile/fenix/pull/11441.
The only issue could be if the buttons are not at all displayed when Talkback is enabled, if so, the title and comment 0 should be updated to reflect the bug.

**Comment by abhijitvalluri** [comment]
Additionally, the bug with the tab + button not being visible in the light theme in private tabs mode is the same as the one reported in #14950, and was fixed in  b0729f6. Please test with that change to verify that the new tab + button is visible in private tabs mode as well.

**Comment by softvision-miralobontiu** [comment]
Closing this issue as a dupe of #14950.
